### PR TITLE
[ticket/13142] Before query to get the total number of reports

### PR DESCRIPTION
**Pull request by brunoais**
Followup to:
https://github.com/phpbb/phpbb/pull/3197

Placing the event in a more meaningful place.

Requires
#3447
to work.

https://tracker.phpbb.com/browse/PHPBB3-13142

### CODE REVIEW

**Comment by brunoais** [comment]
!set event

**Comment by marc1706** [comment]
You need to add $phpbb_dispatcher here: https://github.com/brunoais/phpbb/blob/ticket/13142/phpBB/includes/functions_mcp.php#L371

**Comment by brunoais** [comment]
@marc1706 
> Requires
> #3447
> to work.

**Comment by nickvergessen** [comment]
Reopened to proc travis again

**Comment by brunoais** [comment]
OK.

File: phpBB/includes/functions_mcp.php [hunk 1]
```diff
@@ -575,6 +553,54 @@ function phpbb_mcp_sorting($mode, &$sort_days, &$sort_key, &$sort_dir, &$sort_by
 			break;
 	}
 
+
```
Comment (marc1706) [inline phpBB/includes/functions_mcp.php]: Please remove this extra new line. This will also resolve the travis build fail.

Comment (brunoais) [inline phpBB/includes/functions_mcp.php]: Done.

File: phpBB/includes/functions_mcp.php [hunk 2]
```diff
@@ -606,7 +631,7 @@ function phpbb_mcp_sorting($mode, &$sort_days, &$sort_key, &$sort_dir, &$sort_by
 		$total = (int) $db->sql_fetchfield('total');
 		$db->sql_freeresult($result);
 	}
-	else
+	else if ( $total < -1)
```
Comment (marc1706) [inline phpBB/includes/functions_mcp.php]: Please remove the space before $total

**Pull request merged**